quickget

```diff
@@ -2204,6 +2204,9 @@ function get_opensuse() {
     elif [ "${RELEASE}" == "microos" ]; then
         ISO="openSUSE-MicroOS-DVD-x86_64-Current.iso"
         URL="https://download.opensuse.org/tumbleweed/iso"
+    elif [ "${RELEASE}" == "aeon" ]; then
+        ISO="opensuse-aeon.x86_64.iso"
+        URL="https://mirrorcache-au.opensuse.org/tumbleweed/appliances/iso"
```

Comment:
```suggestion
        URL="https://mirrorcache.opensuse.org/tumbleweed/appliances/iso"
```
Shouldn't this be the URL to enable proper redirection?

Reply:
Whoops, yes it should, copy paste error on my part. Fixing.